pennylane_lightning/core/_measurements_base.py

```diff
@@ -139,15 +140,6 @@ def probs(self, measurementprocess: MeasurementProcess):
         Returns:
             Probabilities of the supplied observable or wires
         """
-        diagonalizing_gates = measurementprocess.diagonalizing_gates()
-        if diagonalizing_gates:
-            self._qubit_state.apply_operations(diagonalizing_gates)
-        results = self._measurement_lightning.probs(measurementprocess.wires.tolist())
-        if diagonalizing_gates:
-            self._qubit_state.apply_operations(
-                [qml.adjoint(g, lazy=False) for g in reversed(diagonalizing_gates)]
-            )
-        return results
```

Comment:
LGPU has different behaviour in `probs` from LQ and LK. LGPU needs to perform an extra conversion of results due to cuQuantum returns as col-major orderings, so performing transpose on data for bit-index shuffle is needed.